share/wake/lib/versions/v34.wake

```diff
@@ -0,0 +1,10 @@
+package v34_wake
```

Comment:
Instead of making a v34 lib, its just as easy to switch packages from v34 to importing plan_scorer so I don't think this v34 buys us anything.